README.md

```diff
@@ -30,7 +30,7 @@ The deployment scheme will be:
 ## Instruction
 
 The repo contain a helper script to perform routine operations.
-Its intended to be "executable documetnation", so the reader is invited to look to the script
+Its intended to be "executable documentation", so the reader is invited to look to the script
```

Comment:
Thanks! Probably it should be even

> It's intended to be "executable documentation", so the reader is invited to look at the script.

And I need to run the whole doc through spell-checker :)